src/otx/algo/classification/model/backbones/otx_efficientnet.py

```diff
@@ -0,0 +1,1303 @@
+        root : str, default '~/.torch/models'. Location for keeping the model parameters.
+        **kwargs: Addition keyword arguments.
+    """
+
+    return get_efficientnet(
+        version="b8",
+        in_size=in_size,
+        tf_mode=True,
+        bn_eps=1e-3,
+        model_name="efficientnet_b8c",
+        **kwargs,
+    )
+
+
+def _calc_width(net):
+    import numpy as np
+
+    net_params = filter(lambda p: p.requires_grad, net.parameters())
+    weight_count = 0
+    for param in net_params:
+        weight_count += np.prod(param.size())
+    return weight_count
+
+
+def _test():
+    import torch
+
+    pretrained = False
+
+    models = [
+        efficientnet_b0,
+        efficientnet_b1,
+        efficientnet_b2,
+        efficientnet_b3,
+        efficientnet_b4,
+        efficientnet_b5,
+        efficientnet_b6,
+        efficientnet_b7,
+        efficientnet_b8,
+        efficientnet_b0b,
+        efficientnet_b1b,
+        efficientnet_b2b,
+        efficientnet_b3b,
+        efficientnet_b4b,
+        efficientnet_b5b,
+        efficientnet_b6b,
+        efficientnet_b7b,
+        efficientnet_b0c,
+        efficientnet_b1c,
+        efficientnet_b2c,
+        efficientnet_b3c,
+        efficientnet_b4c,
+        efficientnet_b5c,
+        efficientnet_b6c,
+        efficientnet_b7c,
+        efficientnet_b8c,
+    ]
+
+    for model in models:
+
+        net = model(pretrained=pretrained)
+
+        # net.train()
+        net.eval()
+        weight_count = _calc_width(net)
+        print(f"m={model.__name__}, {weight_count}")
+        assert model != efficientnet_b0 or weight_count == 5288548
+        assert model != efficientnet_b1 or weight_count == 7794184
+        assert model != efficientnet_b2 or weight_count == 9109994
+        assert model != efficientnet_b3 or weight_count == 12233232
+        assert model != efficientnet_b4 or weight_count == 19341616
+        assert model != efficientnet_b5 or weight_count == 30389784
+        assert model != efficientnet_b6 or weight_count == 43040704
+        assert model != efficientnet_b7 or weight_count == 66347960
+        assert model != efficientnet_b8 or weight_count == 87413142
+        assert model != efficientnet_b0b or weight_count == 5288548
+        assert model != efficientnet_b1b or weight_count == 7794184
+        assert model != efficientnet_b2b or weight_count == 9109994
+        assert model != efficientnet_b3b or weight_count == 12233232
+        assert model != efficientnet_b4b or weight_count == 19341616
+        assert model != efficientnet_b5b or weight_count == 30389784
+        assert model != efficientnet_b6b or weight_count == 43040704
+        assert model != efficientnet_b7b or weight_count == 66347960
+
+        x = torch.randn(1, 3, net.in_size[0], net.in_size[1])
+        y = net(x)
+        y.sum().backward()
+        assert tuple(y.size()) == (1, 1000)
+
+
+@MODELS.register_module()
```

Comment:
This comment is not just for this file but every other files you added for the custom model. I can see that the custom models you added are self-complete and no use of `MMPretrain`'s resource. If this is true, why this should be registered in `MMPretrain.MODELS`?

I hope that you can decouple this such as `OTXEfficientNet(OTXClassificationModel)` in the next PR.

Reply:
Yeah, exactly. My next plan is to convert like what you said

Reply:
Can you add the discussion as a future plan to the Backlog on our OneNote page?

Reply:
> @sungmanc Can you add the discussion as a future plan to the Backlog on our OneNote page?

Added.